shared/team-building/index.tsx

```diff
@@ -857,4 +918,11 @@ const styles = Styles.styleSheetCreate(
     } as const)
 )
 
+const peopleModalProps: Partial<React.ComponentProps<typeof Kb.Modal>> = {
```

Comment:
are these styles something that would make sense to include in `DefaultFullHeight` or a rename of that mode? or do you expect they're one-off for this modal?

Reply:
They're one-off for the people search version of this modal to make it appear over where the search bar is on that page.